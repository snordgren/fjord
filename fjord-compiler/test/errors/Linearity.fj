module errors.Linearity

record Message
  content : String


frugal : Message -* (Message, Message)
frugal m =
  (m, m)

