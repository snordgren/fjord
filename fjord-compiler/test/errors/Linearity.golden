// module errors.Linearity

export const Message = content => {
  var _a = {};
  _a.content = content;
  return _a;
};

export const frugal = m => Object.freeze([m, m]);
